website/contribute/rfc-process.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 3
+title: "RFC Process"
+toc: true
+last_modified_at: 2020-09-01T15:59:57-04:00
+---
+
+This page describes a Request For Comments (RFC) process for proposing any major change to Hudi or even just sharing designs/vision for the project to get early feedback.
+
+## When do you need an RFC ?
```

Comment:
```suggestion
## When do I need an RFC ?
```

Reply:
This is optional..just felt first tense sounds more engaging.. maybe just me.

Reply:
its also me. will fix these. good call